Sources_v3/StreamChatUI/Utils/NSLayoutConstraint+Extensions.swift

```diff
@@ -25,8 +20,13 @@ extension NSLayoutConstraint {
         return self
     }
 
-    /// Returns updated `self` with `priority == .almostRequired`
+    /// Returns updated `self` with `priority == .streamAlmostRequire`
     var almostRequired: NSLayoutConstraint {
-        with(priority: .almostRequired)
+        with(priority: .streamAlmostRequire)
+    }
+
+    /// Default constraint priority
+    var stream: NSLayoutConstraint {
```

Comment:
`NSLayoutConstraint().stream` feels strange. 

What about `NSLayoutConstraint().withStreamPriority` and `NSLayoutConstraint().withAlmostRequiredPriority`. IMO The Apple API for defining constraints `with(priority:) feels really good and those extensions might be not needed... because the apple function is explicit enough. But this is just a tought.